test_vm/src/util.rs

```diff
@@ -580,6 +581,85 @@ pub fn submit_windowed_post(
     .matches(v.take_invocations().last().unwrap());
 }
 
+pub fn change_beneficiary(
+    v: &VM,
+    from: Address,
+    maddr: Address,
+    beneficiary_change_proposal: &ChangeBeneficiaryParams,
+) {
+    apply_ok(
+        v,
+        from,
+        maddr,
+        TokenAmount::zero(),
+        MinerMethod::ChangeBeneficiary as u64,
+        beneficiary_change_proposal.clone(),
+    );
+}
+
+pub fn get_beneficiary(v: &VM, from: Address, m_addr: Address) -> GetBeneficiaryReturn {
+    apply_ok(
+        v,
+        from,
+        m_addr,
+        TokenAmount::zero(),
+        MinerMethod::GetBeneficiary as u64,
+        RawBytes::default(),
+    )
+    .deserialize()
+    .unwrap()
+}
+
+pub fn change_owner_address(v: &VM, from: Address, m_addr: Address, new_miner_addr: Address) {
+    apply_ok(
+        v,
+        from,
+        m_addr,
+        TokenAmount::zero(),
+        MinerMethod::ChangeOwnerAddress as u64,
+        new_miner_addr,
+    );
+}
+
+pub fn withdraw_balance(
+    v: &VM,
+    from: Address,
+    m_addr: Address,
+    to_withdraw_amount: TokenAmount,
+    expect_withdraw_amount: TokenAmount,
+) {
+    let params = WithdrawBalanceParams { amount_requested: to_withdraw_amount };
+    let withdraw_return: WithdrawBalanceReturn = apply_ok(
+        v,
+        from,
+        m_addr,
+        TokenAmount::zero(),
+        MinerMethod::WithdrawBalance as u64,
+        params.clone(),
+    )
+    .deserialize()
+    .unwrap();
+
+    if expect_withdraw_amount.is_positive() {
+        let withdraw_balance_params_se = serialize(&params, "withdraw  balance params").unwrap();
+        ExpectInvocation {
+            from: Some(from),
+            to: m_addr,
+            method: MinerMethod::WithdrawBalance as u64,
+            params: Some(withdraw_balance_params_se),
+            subinvocs: Some(vec![ExpectInvocation {
+                to: from,
```

Comment:
Shouldn't this be the beneficiary address (which might need to be another parameter here)? I guess in the tests so far you are always calling this from the beneficiary address.

Reply:
exit a test for non-beneficiary